### ISSUE TITLE
site logo should have text accessible by screen reader

### ISSUE THREAD

**Issue by Lane** [Hyperobjekt/untd-site#108]
logo should read "NTX Social Mobility Hub" by screen reader (see [making accessible inline svg](https://css-tricks.com/accessible-svgs/#2-inline-svg))
